src/main/java/com/soboleoko/medicalclinic/controller/VisitController.java

```diff
@@ -19,16 +19,16 @@ public class VisitController {
 
     @PostMapping("/{doctorId}")
     public VisitDTO createVisit(@RequestBody @Valid CreateVisitDTO visit, @PathVariable Long doctorId) {
-        return visitMapper.mapToVisitDTO(visitService.createVisit(visit,doctorId));
+        return visitMapper.mapToVisitDTO(visitService.createVisit(visit, doctorId));
     }
 
-    @PatchMapping("/{visitId}/book")
+    @PatchMapping("/{visitId}/book/{patientId}")
```

Comment:
jak chcesz tak zrobić to moim zdaniem lepiej
zeby to bylo np tak: visits/{visitId}/book?patientId=1